Cargo.toml

```diff
@@ -19,9 +19,11 @@ default = ["std"]
 # Activate `std` within the crate. Currently this gives a much faster `sqrt`
 # impl when an explicit hardware sqrt isn't available.
 std = []
+serde = ["dep:serde"]
```

Comment:
oh we need to have this feature removed and then have the `serde` feature just be the optional dep only, at least based on our current MSRV we do.

Reply:
Or rebase forward since this feature was added in Rust 1.60 and that's the new MSRV :)

Reply:
I just rebased so hopefully we should now be good